tests/api/tests.c

```diff
@@ -102,7 +102,9 @@ int main(int argc, char *argv[]) {
    * variable or command-line option.
    */
   srunner_set_log(runner, log_file);
-  srunner_set_xml(runner, xml_file);
+  if (getenv("XML_TEST_OUTPUT")) {
```

Comment:
Thanks!  For consistency, would you object to having this be `PR_XML_TEST_OUTPUT`?